website/client/src/components/ui/selectList.vue

```diff
@@ -53,9 +55,19 @@
 <style lang="scss" scoped>
   @import '~@/assets/scss/colors.scss';
 
+  .select-list .select-list-button-wrap {
+    white-space: nowrap;
+    overflow: hidden;
+    text-overflow: ellipsis;
+  }
+
   .select-list ::v-deep .selectListItem {
```

Comment:
Here you can see the `::v-deep` selector being used, that did not work for you here?

Reply:
It didn't.

I can try again... maybe I might need to do a double v-deep 🤔...

Anyways, wrapping feels nicer XD.